internal/engine/docker_pruner.go

```diff
@@ -0,0 +1,81 @@
+package engine
+
+import (
+	"context"
+	"time"
+
+	"github.com/windmilleng/tilt/internal/docker"
+	"github.com/windmilleng/tilt/internal/store"
+	"github.com/windmilleng/tilt/pkg/logger"
+)
+
+// How often to prune Docker images while Tilt is running
+const dockerPruneInterval = time.Hour * 12
```

Comment:
i wonder if we should do this once on startup and then after every N builds. it seems unlikely that anyone will be running tilt for 12 hours, given an 8 hour workday ever since the labor rights movement in the 30s

i still think it's likely that some people will want this every N builds, or when their disk space is low (see my comment on the other pr)

Reply:
that makes a lot of sense. I'm going to commit this as is (with the pruner disabled) and add that functionality when I make it configurable.